torch/csrc/jit/runtime/calculate_necessary_args.h

```diff
@@ -13,9 +13,19 @@ inline size_t CalculateNecessaryArgs(
   if (schema_args.size() < actual_inputs.size()) {
     return actual_inputs.size();
   }
+
+  int schema_idx = schema_args.size() - 1;
```

Comment:
@iseeyuan what should be the expected `op_to_num_specified_args` with this change? For the following source code
```
class Module(torch.nn.Module):
    def __init__(self):
        super().__init__()
    def forward(self, x, h):
        torch.add(x, h, alpha=2.0, out=x)
```

Should bytecode looks like 
a. 
```
...
('operators', (('aten::add', 'out', 3),))
...
  (('arguments',
    ((('name', 'self'), ('type', '__torch__.m'), ('default_value', None)),
     (('name', 'x'), ('type', 'Tensor'), ('default_value', None)),
     (('name', 'h'), ('type', 'Tensor'), ('default_value', None)))),
```
or 
```
...
('operators', (('aten::add', 'out', 4),))
...
  (('arguments',
    ((('name', 'self'), ('type', '__torch__.m'), ('default_value', None)),
     (('name', 'x'), ('type', 'Tensor'), ('default_value', None)),
     (('name', 'h'), ('type', 'Tensor'), ('default_value', None)))),
```

With this pr, looks like the `op_to_num_specified_args` is 3 now.

Reply:
I think it should be 3. In the .cpp test, if `alpha = 1.0`, `op_to_num_specified_args` = 2.  @tugsbayasgalan could you confirm?

Reply:
I'm checking the bytecode from the source code above, with and without this pr:

With this pr:
![image](https://user-images.githubusercontent.com/16430979/129623334-a6375f00-8a4e-40de-bcb0-8580c497990a.png)


Without this pr:
![image](https://user-images.githubusercontent.com/16430979/129623348-2bb15d3c-0a97-43e5-bdce-d1e66f9e00e3.png)


There are two main differences:
1. the number of specified arguments is different 4->3
2. The instructions are different, we no longer have the `('Move', 2, 0)`. Is it expected?

Another question, for `aten::add` operator, there are four arguments in total. Is it expected? What should be the correct sequence?